src/components/WorkspaceAdd/WorkspaceAddButton.test.tsx

```diff
@@ -58,7 +56,12 @@ describe('Add Workspace modal', () => {
 
   it('Is accessible', async () => {
     openModal();
-    const results = await axe(container);
+    // Button is accessible
+    let results = await axe(screen.getByText('Create workspace'));
+    expect(results).toHaveNoViolations();
```

Comment:
The tests as originally written were evergreen, so this ensures the actual modal is tested